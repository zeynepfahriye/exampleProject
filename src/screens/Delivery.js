import { View, Text, SafeAreaView ,Platform, ScrollView} from 'react-native'
import React from 'react'
import Header from '../components/Header'
import { Avatar, Card, List } from 'react-native-paper'
import MapView from 'react-native-maps'

const Delivery = ({navigation}) => {
  return (
    <SafeAreaView style={{flex:1}}>
    {Platform.OS === 'ios' &&
    <View style={{
        width: "100%",
        height: 100, // For all devices, even X, XS Max
        position: 'absolute',
        top: 0,
        left: 0,
        backgroundColor: "orange"}}
   />}
    <Header title={"Delivery Details"} hasBackButton={true} navigation={navigation}/>
      <View style={{flex:1}}>
        <MapView
        style={{flex:1}}
        initialRegion={{
          latitude: -3.722,
          longitude: -38.515,
          latitudeDelta: 0.09,
          longitudeDelta: 0.04,
        }}>
        </MapView>
      </View>
      <Card>
        <Card.Title 
            title={"01/01/1111"}
            titleStyle={{fontWeight:'bold'}}
            right={()=>
            <Text style={{marginHorizontal:10,fontWeight:'bold',color:'purple'}}>$30,00</Text>
            }
        />
        <Card.Content>
            <List.Item
            title={"Zeynep Çakır"}
            description="53 deliveries"
            left={()=>
            <Avatar.Image
            size={52}
            source={require('../assets/icons/woman-2.png')}
             />
            }
            />
            <List.Item
            title="Origin"
            description="Origin street,40"
             left={()=>
             <List.Icon icon="flag"/>}
            />
            <List.Item
            title="Origin"
            description="Origin street,40"
             left={()=>
             <List.Icon icon="flag-checkered"/>}
            />
        </Card.Content>
      </Card>
    </SafeAreaView>
  )
}

export default Delivery