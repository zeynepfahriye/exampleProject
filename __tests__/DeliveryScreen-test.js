import React from 'react'
import 'react-native'
import renderer from 'react-test-renderer'
import Delivery from '../src/screens/Delivery'

test('Form header snapshot',()=>{
    const snap = renderer.create(
        <Delivery />
    ).toJSON();
    expect(snap).toMatchSnapshot();
})