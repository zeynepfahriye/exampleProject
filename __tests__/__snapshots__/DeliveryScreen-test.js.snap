// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`Form header snapshot 1`] = `
<RCTSafeAreaView
  style={
    Object {
      "flex": 1,
    }
  }
>
  <View
    style={
      Object {
        "backgroundColor": "orange",
        "height": 100,
        "left": 0,
        "position": "absolute",
        "top": 0,
        "width": "100%",
      }
    }
  />
  <View
    collapsable={false}
    style={
      Object {
        "alignItems": "center",
        "backgroundColor": "orange",
        "elevation": 4,
        "flexDirection": "row",
        "height": 56,
        "paddingHorizontal": 4,
        "shadowColor": "#000000",
        "shadowOffset": Object {
          "height": 3,
          "width": 0,
        },
        "shadowOpacity": 0.24,
        "shadowRadius": 4,
      }
    }
  >
    <View
      accessibilityLabel="Back"
      accessibilityRole="button"
      accessibilityState={
        Object {
          "disabled": false,
        }
      }
      accessible={true}
      focusable={true}
      hitSlop={
        Object {
          "bottom": 6,
          "left": 6,
          "right": 6,
          "top": 6,
        }
      }
      onClick={[Function]}
      onResponderGrant={[Function]}
      onResponderMove={[Function]}
      onResponderRelease={[Function]}
      onResponderTerminate={[Function]}
      onResponderTerminationRequest={[Function]}
      onStartShouldSetResponder={[Function]}
      style={
        Array [
          Object {
            "overflow": "hidden",
          },
          Array [
            Object {
              "alignItems": "center",
              "justifyContent": "center",
              "margin": 6,
              "overflow": "hidden",
            },
            Object {
              "borderRadius": 18,
              "height": 36,
              "width": 36,
            },
            undefined,
            undefined,
          ],
        ]
      }
    >
      <View>
        <View
          style={
            Array [
              Object {
                "alignItems": "center",
                "justifyContent": "center",
              },
              Object {
                "height": 24,
                "width": 24,
              },
            ]
          }
        >
          <View
            collapsable={false}
            style={
              Object {
                "bottom": 0,
                "left": 0,
                "opacity": 1,
                "position": "absolute",
                "right": 0,
                "top": 0,
                "transform": Array [
                  Object {
                    "rotate": "0deg",
                  },
                ],
              }
            }
          >
            <View
              style={
                Array [
                  Object {
                    "alignItems": "center",
                    "justifyContent": "center",
                  },
                  Object {
                    "height": 24,
                    "transform": Array [
                      Object {
                        "scaleX": 1,
                      },
                    ],
                    "width": 24,
                  },
                ]
              }
            >
              <Image
                source={
                  Object {
                    "testUri": "../../react-native-paper/lib/commonjs/assets/back-chevron.png",
                  }
                }
                style={
                  Array [
                    Object {
                      "resizeMode": "contain",
                    },
                    Object {
                      "height": 21,
                      "tintColor": "white",
                      "width": 21,
                    },
                  ]
                }
              />
            </View>
          </View>
        </View>
      </View>
    </View>
    <View
      accessibilityState={
        Object {
          "disabled": true,
        }
      }
      accessible={true}
      focusable={false}
      onClick={[Function]}
      onResponderGrant={[Function]}
      onResponderMove={[Function]}
      onResponderRelease={[Function]}
      onResponderTerminate={[Function]}
      onResponderTerminationRequest={[Function]}
      onStartShouldSetResponder={[Function]}
      style={
        Array [
          Object {
            "flex": 1,
            "paddingHorizontal": 12,
          },
          Array [
            Object {
              "marginLeft": 8,
            },
            Object {
              "alignItems": "center",
            },
            undefined,
          ],
        ]
      }
    >
      <Text
        accessibilityRole="header"
        accessibilityTraits="header"
        accessible={true}
        numberOfLines={1}
        style={
          Array [
            Object {
              "color": "#000000",
              "fontFamily": "System",
              "fontWeight": "400",
            },
            Object {
              "textAlign": "left",
            },
            Array [
              Object {
                "color": "white",
                "fontFamily": "System",
                "fontWeight": "400",
              },
              Object {
                "fontSize": 17,
              },
              undefined,
            ],
          ]
        }
      >
        Delivery Details
      </Text>
    </View>
    <View
      style={
        Object {
          "width": 48,
        }
      }
    />
  </View>
  <View
    style={
      Object {
        "flex": 1,
      }
    }
  >
    <AIRMap
      initialRegion={
        Object {
          "latitude": -3.722,
          "latitudeDelta": 0.09,
          "longitude": -38.515,
          "longitudeDelta": 0.04,
        }
      }
      onChange={[Function]}
      onMapReady={[Function]}
      region={null}
      style={
        Object {
          "flex": 1,
        }
      }
    />
  </View>
  <View
    collapsable={false}
    style={
      Object {
        "backgroundColor": "#ffffff",
        "borderColor": "rgba(0, 0, 0, 0.12)",
        "borderRadius": 4,
        "elevation": 1,
        "shadowColor": "#000000",
        "shadowOffset": Object {
          "height": 0.5,
          "width": 0,
        },
        "shadowOpacity": 0.24,
        "shadowRadius": 0.75,
      }
    }
  >
    <View
      accessibilityState={
        Object {
          "disabled": true,
        }
      }
      accessible={true}
      focusable={false}
      onClick={[Function]}
      onResponderGrant={[Function]}
      onResponderMove={[Function]}
      onResponderRelease={[Function]}
      onResponderTerminate={[Function]}
      onResponderTerminationRequest={[Function]}
      onStartShouldSetResponder={[Function]}
      style={
        Object {
          "flexGrow": 1,
          "flexShrink": 1,
        }
      }
    >
      <View
        style={
          Array [
            Object {
              "alignItems": "center",
              "flexDirection": "row",
              "justifyContent": "space-between",
              "paddingLeft": 16,
            },
            Object {
              "minHeight": 72,
            },
            undefined,
          ]
        }
      >
        <View
          style={
            Array [
              Object {
                "flex": 1,
                "flexDirection": "column",
                "justifyContent": "center",
              },
            ]
          }
        >
          <Text
            numberOfLines={1}
            style={
              Array [
                Object {
                  "color": "#000000",
                  "fontFamily": "System",
                  "fontWeight": "400",
                },
                Object {
                  "textAlign": "left",
                },
                Array [
                  Object {
                    "textAlign": "left",
                  },
                  Object {
                    "color": "rgba(0, 0, 0, 0.87)",
                    "fontFamily": "System",
                    "fontWeight": "500",
                    "writingDirection": "ltr",
                  },
                  Array [
                    Object {
                      "fontSize": 20,
                      "letterSpacing": 0.15,
                      "lineHeight": 30,
                      "marginVertical": 2,
                    },
                    Array [
                      Object {
                        "minHeight": 30,
                        "paddingRight": 16,
                      },
                      Object {
                        "marginBottom": 2,
                      },
                      Object {
                        "fontWeight": "bold",
                      },
                    ],
                  ],
                ],
              ]
            }
          >
            01/01/1111
          </Text>
        </View>
        <View>
          <Text
            style={
              Object {
                "color": "purple",
                "fontWeight": "bold",
                "marginHorizontal": 10,
              }
            }
          >
            $30,00
          </Text>
        </View>
      </View>
      <View
        style={
          Array [
            Object {
              "paddingHorizontal": 16,
            },
            Object {
              "paddingBottom": 16,
            },
            undefined,
          ]
        }
      >
        <View
          accessibilityState={
            Object {
              "disabled": true,
            }
          }
          accessible={true}
          focusable={false}
          onClick={[Function]}
          onResponderGrant={[Function]}
          onResponderMove={[Function]}
          onResponderRelease={[Function]}
          onResponderTerminate={[Function]}
          onResponderTerminationRequest={[Function]}
          onStartShouldSetResponder={[Function]}
          style={
            Array [
              false,
              Array [
                Object {
                  "padding": 8,
                },
                undefined,
              ],
            ]
          }
        >
          <View
            style={
              Object {
                "flexDirection": "row",
              }
            }
          >
            <View
              style={
                Array [
                  Object {
                    "backgroundColor": "#6200ee",
                    "borderRadius": 26,
                    "height": 52,
                    "width": 52,
                  },
                  undefined,
                ]
              }
            >
              <Image
                source={
                  Object {
                    "testUri": "../../../src/assets/icons/woman-2.png",
                  }
                }
                style={
                  Object {
                    "borderRadius": 26,
                    "height": 52,
                    "width": 52,
                  }
                }
              />
            </View>
            <View
              style={
                Array [
                  Object {
                    "marginVertical": 6,
                    "paddingLeft": 8,
                  },
                  Object {
                    "flex": 1,
                    "justifyContent": "center",
                  },
                ]
              }
            >
              <Text
                numberOfLines={1}
                selectable={false}
                style={
                  Array [
                    Object {
                      "color": "#000000",
                      "fontFamily": "System",
                      "fontWeight": "400",
                    },
                    Object {
                      "textAlign": "left",
                    },
                    Array [
                      Object {
                        "fontSize": 16,
                      },
                      Object {
                        "color": "rgba(0, 0, 0, 0.87)",
                      },
                      undefined,
                    ],
                  ]
                }
              >
                Zeynep Çakır
              </Text>
              <Text
                numberOfLines={2}
                selectable={false}
                style={
                  Array [
                    Object {
                      "color": "#000000",
                      "fontFamily": "System",
                      "fontWeight": "400",
                    },
                    Object {
                      "textAlign": "left",
                    },
                    Array [
                      Object {
                        "fontSize": 14,
                      },
                      Object {
                        "color": "rgba(0, 0, 0, 0.54)",
                      },
                      undefined,
                    ],
                  ]
                }
              >
                53 deliveries
              </Text>
            </View>
          </View>
        </View>
        <View
          accessibilityState={
            Object {
              "disabled": true,
            }
          }
          accessible={true}
          focusable={false}
          onClick={[Function]}
          onResponderGrant={[Function]}
          onResponderMove={[Function]}
          onResponderRelease={[Function]}
          onResponderTerminate={[Function]}
          onResponderTerminationRequest={[Function]}
          onStartShouldSetResponder={[Function]}
          style={
            Array [
              false,
              Array [
                Object {
                  "padding": 8,
                },
                undefined,
              ],
            ]
          }
        >
          <View
            style={
              Object {
                "flexDirection": "row",
              }
            }
          >
            <View
              pointerEvents="box-none"
              style={
                Array [
                  Object {
                    "alignItems": "center",
                    "height": 40,
                    "justifyContent": "center",
                    "margin": 8,
                    "width": 40,
                  },
                  undefined,
                ]
              }
            >
              <Text
                accessibilityElementsHidden={true}
                allowFontScaling={false}
                importantForAccessibility="no-hide-descendants"
                pointerEvents="none"
                selectable={false}
                style={
                  Array [
                    Object {
                      "color": "#000000",
                      "fontSize": 24,
                    },
                    Array [
                      Object {
                        "lineHeight": 24,
                        "transform": Array [
                          Object {
                            "scaleX": 1,
                          },
                        ],
                      },
                      Object {
                        "backgroundColor": "transparent",
                      },
                    ],
                    Object {
                      "fontFamily": "Material Design Icons",
                      "fontStyle": "normal",
                      "fontWeight": "normal",
                    },
                    Object {},
                  ]
                }
              >
                󰈻
              </Text>
            </View>
            <View
              style={
                Array [
                  Object {
                    "marginVertical": 6,
                    "paddingLeft": 8,
                  },
                  Object {
                    "flex": 1,
                    "justifyContent": "center",
                  },
                ]
              }
            >
              <Text
                numberOfLines={1}
                selectable={false}
                style={
                  Array [
                    Object {
                      "color": "#000000",
                      "fontFamily": "System",
                      "fontWeight": "400",
                    },
                    Object {
                      "textAlign": "left",
                    },
                    Array [
                      Object {
                        "fontSize": 16,
                      },
                      Object {
                        "color": "rgba(0, 0, 0, 0.87)",
                      },
                      undefined,
                    ],
                  ]
                }
              >
                Origin
              </Text>
              <Text
                numberOfLines={2}
                selectable={false}
                style={
                  Array [
                    Object {
                      "color": "#000000",
                      "fontFamily": "System",
                      "fontWeight": "400",
                    },
                    Object {
                      "textAlign": "left",
                    },
                    Array [
                      Object {
                        "fontSize": 14,
                      },
                      Object {
                        "color": "rgba(0, 0, 0, 0.54)",
                      },
                      undefined,
                    ],
                  ]
                }
              >
                Origin street,40
              </Text>
            </View>
          </View>
        </View>
        <View
          accessibilityState={
            Object {
              "disabled": true,
            }
          }
          accessible={true}
          focusable={false}
          onClick={[Function]}
          onResponderGrant={[Function]}
          onResponderMove={[Function]}
          onResponderRelease={[Function]}
          onResponderTerminate={[Function]}
          onResponderTerminationRequest={[Function]}
          onStartShouldSetResponder={[Function]}
          style={
            Array [
              false,
              Array [
                Object {
                  "padding": 8,
                },
                undefined,
              ],
            ]
          }
        >
          <View
            style={
              Object {
                "flexDirection": "row",
              }
            }
          >
            <View
              pointerEvents="box-none"
              style={
                Array [
                  Object {
                    "alignItems": "center",
                    "height": 40,
                    "justifyContent": "center",
                    "margin": 8,
                    "width": 40,
                  },
                  undefined,
                ]
              }
            >
              <Text
                accessibilityElementsHidden={true}
                allowFontScaling={false}
                importantForAccessibility="no-hide-descendants"
                pointerEvents="none"
                selectable={false}
                style={
                  Array [
                    Object {
                      "color": "#000000",
                      "fontSize": 24,
                    },
                    Array [
                      Object {
                        "lineHeight": 24,
                        "transform": Array [
                          Object {
                            "scaleX": 1,
                          },
                        ],
                      },
                      Object {
                        "backgroundColor": "transparent",
                      },
                    ],
                    Object {
                      "fontFamily": "Material Design Icons",
                      "fontStyle": "normal",
                      "fontWeight": "normal",
                    },
                    Object {},
                  ]
                }
              >
                󰈼
              </Text>
            </View>
            <View
              style={
                Array [
                  Object {
                    "marginVertical": 6,
                    "paddingLeft": 8,
                  },
                  Object {
                    "flex": 1,
                    "justifyContent": "center",
                  },
                ]
              }
            >
              <Text
                numberOfLines={1}
                selectable={false}
                style={
                  Array [
                    Object {
                      "color": "#000000",
                      "fontFamily": "System",
                      "fontWeight": "400",
                    },
                    Object {
                      "textAlign": "left",
                    },
                    Array [
                      Object {
                        "fontSize": 16,
                      },
                      Object {
                        "color": "rgba(0, 0, 0, 0.87)",
                      },
                      undefined,
                    ],
                  ]
                }
              >
                Origin
              </Text>
              <Text
                numberOfLines={2}
                selectable={false}
                style={
                  Array [
                    Object {
                      "color": "#000000",
                      "fontFamily": "System",
                      "fontWeight": "400",
                    },
                    Object {
                      "textAlign": "left",
                    },
                    Array [
                      Object {
                        "fontSize": 14,
                      },
                      Object {
                        "color": "rgba(0, 0, 0, 0.54)",
                      },
                      undefined,
                    ],
                  ]
                }
              >
                Origin street,40
              </Text>
            </View>
          </View>
        </View>
      </View>
    </View>
  </View>
</RCTSafeAreaView>
`;
